dash-pipeline/bmv2/stages/outbound_routing.p4

```diff
@@ -6,12 +6,29 @@
 control outbound_routing_stage(inout headers_t hdr,
                                inout metadata_t meta)
 {
+
+    action set_routing_group_version(bit<1> admin_state) {
```

Comment:
since the parameter is not a version, do you mind to change this to `set_routing_group_attr`?

Reply:
also do you mind to change admin_state to disabled?

there are 2 reasons:
- disabled is more intuitive than admin_state.
- admin_state should not be frequently used (otherwise, why programming it), hence it will be better to make use of the default value. 

Making this field disabled with default value set to false seems to be a good way to achieve these 2 things.

Reply:
Done